commands/adminsonline.js

```diff
@@ -22,18 +43,32 @@ module.exports = {
 
             rcon.on("connect", () => console.log("connected"));
             rcon.on("authenticated", () => console.log("authenticated"));
+            rcon.on("error", () => console.log("errors"));
```

Comment:
lol you shood do `
```suggestion
            rcon.on("error", (err) => console.log(err));
```
as this will ouput the error which is usefull to know

Reply:
right now on the rcon lib we use no error ever exist. this is code that "should" work and  be put back in the lib at some point. but is still a good point